doc/how_to_upgrade_builders.rst

```diff
@@ -76,7 +77,9 @@ Repeat the previous steps.
 
 The remaining step is to configure ``copr_builder_images.aws.{aarch64,x86_64}``
 options in `Ansible git repo`_, in file ``inventory/group_vars/copr_dev_aws``
-and reprovision the ``copr-be-dev`` instance, see :ref:`Testing`.
+and reprovision the ``copr-be-dev`` instance, see :ref:`Testing`.  Document the
```

Comment:
I suppose "Document" means adjusting this comment?

    # n-1: aarch64: ami-0f93da3f4f368afb4
    # n-1: x86_64: ami-062f0015f85924f2f
    # n-0: aarch64: ami-0a2abb60d2e09f6fa
    # n-0: x86_64: ami-0460b8805bf9781cf

`n-0` is confusing because `n-0 = n` ...

Reply:
I jumped with the comment too early. I am now reading the next line which says 

    previous ``ami-*`` variants as ``n-1``

but I would expect to take the current `x86_64: ami-0c430e4bc139d93ec` put the new value there, and move the old one to `n-0: x86_64` not `n-1` like you are suggesting.

Reply:
The `n-0` variants are just confusing, I'm going to remove those right away.  I just commented images I generated last week so I don't forget to remove them.  And then I forgot to drop the comment.